base_delivery_carrier_files/test/carrier_file.yml

```diff
@@ -15,7 +15,8 @@
 -
   I confirm outgoing shipment of 130 kgm Ice-cream.
 -
-  !workflow {model: stock.picking, action: button_confirm, ref: outgoing_shipment_carrier_file}
+  !python {model: stock.picking}: |
+    self.action_confirm(cr, uid, ref("outgoing_shipment_carrier_file"))
```

Comment:
You can use
`!python {model: stock.picking, id: <xml ref>}: |`

Then you can use self as in new API:
`self.action_confirm()`